File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> mp;
        for(int i = 0; i < nums.size(); i++){
            mp[nums[i]] = i + 1;
        }
        vector<int> ans(2);
        for (int i = 0; i < nums.size(); i++) {
            if (mp.contains(target-nums[i]) && mp[target-nums[i]] != i+1) {
                ans[0] = i;
                ans[1] = mp[target-nums[i]] - 1;
            }
        }
        sort(ans.begin(), ans.end());
        return ans;
    }
};

File: Data Structures & Algorithms/car-fleet/submission-1.py
class Solution:
    def carFleet(self, target: int, position: List[int], speed: List[int]) -> int:
        n = len(position)
        vec = []
        for i in range(n):
            vec.append((position[i], speed[i]))
        vec.sort(reverse=True)
        stack = []
        for i in range(n):
            x,y = vec[i]
            time = (target-x)/y
            print(time)
            if len(stack) == 0 or time > stack[len(stack)-1]:
                stack.append(time)
        print(stack)
        return len(stack)
        